vite.config.ts

```diff
@@ -1,10 +1,19 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
+import svgr from "vite-plugin-svgr";
 // import tsconfigPaths from "vite-tsconfig-paths";
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
+  plugins: [
+    react(),
+    svgr({
+      svgrOptions: {
+        icon: true,
+        memo: true,
+      },
+    }),
```

Comment:
```suggestion
    svgr({
       svgrOptions: {
         icon: true,
         memo: true,
       },
       include: '**/*.svg',
     }),
``` 

`include: '**/*.svg' 추가해주면 `?react` 추가 없이 사용 가능한걸로 알아요 ㅎㅎ

Reply:
헉..!! 알려주셔서 감사합니다 ㅎㅎ!!